### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
Added Badge component to component library.

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    width: 300px;
+    text-align: center;
+    font-weight: var(--bold-weight);
+}
+
+.alert-error {
+    background-color: var(--primary-color);
+}
+
+.alert-success {
```
Comment (GuptaEesh) [inline components/alert/alert.css:19]: use css variables

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    background-color: #00FF00;
+}
+
+.alert-primary {
+    background-color: #961ee1;
```
Comment (GuptaEesh) [inline components/alert/alert.css]: use css variables

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.alert-warning {
+    background-color: #d3e916;
```
Comment (GuptaEesh) [inline components/alert/alert.css]: use css variables

**Review by GuptaEesh — COMMENTED** [review]
Rest everything looks good. Just added a few comments.

File: components/alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@import "../../css/variables.css";
+
+.alert {
+    margin: 1rem;
+    padding: 1rem;
+    border: 1px solid white;
+    background-color: white;
+    color: black;
```
Comment (ZuberDunge) [inline components/alert/alert.css]: try to use css variables, it's a good practice.

File: components/button/button.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+.button {
+    border-radius: 40px;
+    border: none;
+    color: black;
+    padding: 1rem;
+    margin: 1rem;
+    cursor: pointer;
+}
+
+a {
```
Comment (ZuberDunge) [inline components/button/button.css]: you can add this in reset.css.

**Review by ZuberDunge — APPROVED** [review]
Added few suggestions, apart from that, rest looks good. Good to go from my side.

**Comment by theprogrammerinyou** [comment]
Done with 2 reviews, also done with suggestions. Closing this PR.

**Pull request closed without merging**